src/rpc/rawtransaction.cpp

```diff
@@ -137,7 +137,7 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
 
     // Accept either a bool (true) or a num (>=1) to indicate verbose output.
     bool fVerbose = false;
-    if (request.params.size() > 1) {
+    if (!request.params[1].isNull()) {
```

Comment:
This could be simplified to:
```cpp
bool fVerbose = false;
if (request.params[1].isNum()) {
    fVerbose = request.params[1].get_int() != 0;
} else if (!request.params[1].isNull()) {
    fVerbose = request.params[1].get_bool();
}
```
With the advantage of removing custom error handling.